client/push/sw_manager.js

```diff
@@ -17,6 +40,18 @@
     }
 
     function sendPushTokenToServer() {
+      var http = new XMLHttpRequest();
+      var url = "https://localhost:9443/api/push/client";
```

Comment:
Lets add "TODO : " to make sure we will change this url from config or running server url